src/components/icon-tile/style.module.css

```diff
@@ -58,9 +58,9 @@ but is necessary for us to implement the current designs for the
 /sentinel and /hcp product landing pages.
 */
 .color-neutral-dark {
-  --icon-color: var(--token-color-neutral-background-primary);
-  --border-color: var(--token-color-neutral-foreground-primary);
-  --background: var(--token-color-neutral-foreground-secondary);
+  --icon-color: var(--token-color-foreground-high-contrast);
```

Comment:
Since @didoo has wrapped up his day, I thought I'd chime in. 

Surface: Use for component and container backgrounds. 
Foreground: Use for text and icons.

Since this appears to be for an icon, it should be using a `Foreground` value. 

More details can be found in the docs: https://design-system-components-hashicorp.vercel.app/foundations/colors 🙂

Reply:
Yes, exactly: what @heatherlarsen said.